src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/github/v4/GraphqlCheckRunProvider.java

```diff
@@ -178,29 +180,36 @@ public DecorationResult createCheckRun(AnalysisDetails analysisDetails, AlmSetti
 
     }
 
-    private void postSummaryComment(String apiUrl, Map<String, String> headers, String projectPath, String pullRequestKey, String summary) throws IOException {
+    void postSummaryComment(String graphqlUrl, Map<String, String> headers, String projectPath, String pullRequestKey, String summary) throws IOException {
```

Comment:
If this has been made visible purely for testing, then it indicate this class has potentially got too complex. Is it worth moving this into a `SummaryCommentHandler` class?